src/core/js/models/adaptModel.js

```diff
@@ -103,18 +109,18 @@ define([
         // Check that property is trackable
         var trackablePropertyNames = _.result(this, 'trackable', []);
         var changedPropertyNames = _.keys(model.changed);
```

Comment:
Is it possible to switch to `const` (and `let` where needed) throughout?

Also, suggest native method:
```suggestion
        const changedPropertyNames = Object.keys(model.changed);
```